doc/source/comparison_with_sql.rst

```diff
@@ -372,10 +372,123 @@ In pandas, you can use :meth:`~pandas.concat` in conjunction with
 
     pd.concat([df1, df2]).drop_duplicates()
 
+SOME ANALYTIC AND AGGREGATE FUNCTIONS
+-------------------------------------
+Top N rows with offset
+
+.. code-block:: sql
+
+    -- MySQL
```

Comment:
you only need 1 sql example. prefer idiomatic sql, prefer (postgres,sqlite,mysql) syntax over oracle (as its a bit non-standard)